deployment/run.sh

```diff
@@ -30,11 +30,27 @@ case "${mode}" in
   install)
     install_logging
     ;;
+  uninstall)
+    delete_logging
+    ;;
+  reinstall)
+    delete_logging
+    install_logging
+    ;;
   migrate)
```

Comment:
in my mind they were still two different things... but we can probably just combine it.